tests/performance/delivery/buckets/prototype/db_buckets.php

```diff
@@ -54,7 +55,7 @@ function createTableFromQuery($tableName, $query, $pk)
         } else {
             $query = str_replace('{pk}', '', $query);
         }
-        if ($this->dbType == 'mysql') {
+        if ($this->dbType == 'mysql' || $this->dbType == 'mysqli') {
```

Comment:
Hmm, maybe add a wrapper for all similar codes like `OA_isMySqlDatabaseType($this->dbType)` ?